helper/plancheck/expect_empty_plan_test.go

```diff
@@ -0,0 +1,79 @@
+package plancheck
```

Comment:
For new functionality, it might be good to implement this as a test package for [black-box testing](https://pkg.go.dev/testing):

```suggestion
package plancheck_test
```

This will require importing the "regular" package code but does two things:

- Ensures the exported interfaces have everything a consumer would need since you would also need them for testing
- Prevents unexpectedly exporting testing-only functionality

Maybe in the future some the Go module handling can differentiate testing-only imports too like how NodeJS has dev packages. 😄

Reply:
Good call, fixed ⚡